Plutarch/DataRepr/Internal/Field.hs

```diff
@@ -100,6 +122,41 @@ instance
   type PFields (PAsData a) = PFields a
   ptoFields = ptoFields . pfromData
 
+
+{- | 
+    always makes sure the result is not wrapped in PAsData, 
+    even if the result it not originally wrapped into it 
+-}
+class PNoAsData (a::PType) (b::PType) | a -> b where 
```

Comment:
Where is this used?

Reply:
nowhere yet but I asked @L-as and he said that it might be useful somewhere in the future. it's basically the other class but vice versa.